src/Install.php

```diff
@@ -137,7 +137,25 @@ public static function check_version() {
 		 */
 		if ( ! $version_option || $requires_update ) {
 			self::install();
+			/**
+			 * WooCommerce Admin has been installed or updated.
+			 */
 			do_action( 'woocommerce_admin_updated' );
```

Comment:
The `woocommerce_admin_updated` action gets triggered on install or update, so I had to add some more specific actions below - `woocommerce_admin_updated_existing` is what is used in this particular note.